### DISCUSSION TITLE
interface is bugged

### DISCUSSION THREAD

**Discussion by eclairvanna** [atom/atom#23150] [Q&A]
hello there! when i was about to save a script (CSS to be specific) the interface is bugged and i have no idea how to turn it back 
![gambar](https://user-images.githubusercontent.com/93168865/138791099-be9a10b1-de3c-4467-90cb-baeb08407a01.png)
is there any way to fix this? sorry i'm not entirely familiar with atom "^^

**Comment by eclairvanna** [comment]
no one?

**Reply by icecream17** [reply]
I guess you could try `ctrl+shift+i` and screenshot what happens?

**Reply by eclairvanna** [reply]
hi! apologies for the overdue reply :( i tried that command in this shows up 
![gambar](https://user-images.githubusercontent.com/93168865/147997868-a369ba6a-f04e-4191-80b6-3892d9e0cf26.png)

**Reply by icecream17** [reply]
Hmm, there's an error that I don't recognize. Don't know where that comes from, maybe a bugged package or something?????

If it's from a package you could run `atom --safe` in cmd and see if atom works.

Could you try "Inspect Element" on those cyan things? (Click at 1 then 2)
![image](https://user-images.githubusercontent.com/58114641/148127239-8fba6bc7-67d2-4723-8c63-37a0b245b70a.png)

**Reply by eclairvanna** [reply]
is this correct?
![gambar](https://user-images.githubusercontent.com/93168865/148141762-8ae00e8d-7013-4f54-802b-59851bf553fe.png)

**Reply by icecream17** [reply]
Yes

<details>
<summary>It seems like these are supposed to be regular panes</summary>

![image](https://user-images.githubusercontent.com/58114641/148145081-62dab8a8-745b-4952-9466-643ec5d8c8a8.png)
![image](https://user-images.githubusercontent.com/58114641/148145565-6c56431a-934c-4eaf-80d5-cc4ad2a4209e.png)

(for me, the rightmost one is a hidden git popup thing)

</details>

It's weird that `div` has all those styles. That's probably some weird theme (or package?).

When I added those styles to `div` the screen looks like this:
![image](https://user-images.githubusercontent.com/58114641/148146027-fc13689c-e162-40a7-9f2c-4eea852d24f9.png)

__Maybe try disabling those weird styles (temporarily)__
When you hover around that rectangle of whitespace, some checkboxes will appear. If you deactivate the checkboxes under `div` then the visual appearance will probably be much less crazy:
![image](https://user-images.githubusercontent.com/58114641/148146193-c300356f-7525-400c-bd84-426f83e9e59e.png)

Example of deactivated checkboxes (middle-top right, one checkbox is still activated):
![image](https://user-images.githubusercontent.com/58114641/148146545-781e7e7f-7d18-4f61-8157-9004d6ae6dae.png)

Once it looks normal enough, you can try disabling any themes/packages you've installed in the settings. Or maybe the css you saved was the atom customization stylesheet? Anyways if this doesn't work you can always screenshot.

<details><summary>I tried adding that div stuff to my styles</summary>

...and disabled it in the console. But for some reason some things are still weird:

![image](https://user-images.githubusercontent.com/58114641/148147005-ee2b2653-a236-4491-be51-6ed57cf8708a.png)

Woah, I switched to "settings" and back to the styles and it looks normal????
![image](https://user-images.githubusercontent.com/58114641/148147360-2df8d950-abe1-421d-8336-fc25f500eee9.png)


</details>
